cli/script/command-executor.ts

```diff
@@ -908,6 +908,12 @@ function tryBase64Decode(encoded: string): string {
     }
 }
 
+function isBinaryOrZip(path: string): boolean {
+    return path.search(/\.zip$/i) !== -1
+    || path.search(/\.apk$/i) !== -1
+    || path.search(/\.ipa$/i) !== -1;
```

Comment:
I see.. I thought you meant remove 8 spaces of indentation that I had previous to this change.